index.js

```diff
@@ -84,6 +84,27 @@ exports.zeros = function (bytes) {
 }
 
 /**
+  * Returns a zero address
+  * @method zeroAddress
+  * @return {Buffer}
+  */
+exports.zeroAddress = function () {
```

Comment:
Though this is also something ``zero`` related, functions are up-till-now sorted by ``Buffer`` related and ``Address`` related so can you move these down to the ``Address`` functions, e.g. after ``isValidAddress``?